spec/controllers/idv/enter_password_controller_spec.rb

```diff
@@ -263,7 +263,7 @@ def show
     end
 
     it 'redirects to phone step if the user has not completed it' do
-      subject.idv_session.user_phone_confirmation = nil
+      subject.idv_session.vendor_phone_confirmation = nil
```

Comment:
This was just bad setup. vendor_phone_confirmation is set by the phone step, user_phone_confirmation is set by the phone_confirmation step.